t/storage.t

```diff
@@ -179,42 +141,107 @@ describe "HTFeed::Storage" => sub {
           ok($storage->{is_repeat});
         }
       };
+    };
 
-      context "when link path is the same as object path" => sub {
-        it "works" => sub {
-          my $storage = nonlinked_storage('test','test');
-          $storage->make_object_path;
+    describe "#stage" => sub {
+      context "when the item is in the repository with a different storage path" => sub {
+        it "deposits to a staging area under that path" => sub {
+          make_old_version_other_dir('test','test');
+          my $storage = linked_storage( 'test', 'test');
+          $storage->stage;
 
-          ok(-e "$tmpdirs->{link_dir}/test/pairtree_root/te/st/test");
+          ok(-e "$tmpdirs->{other_obj_dir}/.tmp/test.test/test.mets.xml");
+          ok(-e "$tmpdirs->{other_obj_dir}/.tmp/test.test/test.zip");
         };
+      };
+    };
+  };
 
-        it "is idempotent" => sub {
+  describe "HTFeed::Storage::LocalPairtree" => sub {
+    use HTFeed::Storage::LocalPairtree;
+
+    sub nonlinked_storage {
```

Comment:
should rename this to "local_storage"

Reply:
Agree, "nonlinked" makes me think of `unlink` command.